shopping_mall/backend/tests/agent/test_executor.py

```
@@ -0,0 +1,849 @@
+        assert "배송" in result.answer
+        assert result.escalated is False
+        assert len(fallback.calls) == 2     # fallback이 처음부터 루프 실행
+
+
+# ══════════════════════════════════════════════════════════════════
+# intent 역산 로직
+# ══════════════════════════════════════════════════════════════════
+
+class TestIntentMapping:
+
+    @pytest.mark.parametrize("tool_name,expected_intent", [
```

Ruff (0.15.9)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)